cli/src/main/java/org/jboss/as/cli/handlers/OperationRequestHandler.java

```diff
@@ -174,7 +173,8 @@ private ModelNode validateRequest(CommandContext ctx, ModelNode request) throws
             throw new CommandFormatException("Failed to perform " + Util.READ_OPERATION_DESCRIPTION + " to validate the request: result is not available.");
         }
         final ModelNode result = outcome.get(Util.RESULT);
-        if(!result.hasDefined(Util.REQUEST_PROPERTIES)) {
+        final Set<String> definedProps = result.get(Util.REQUEST_PROPERTIES).keys();
```

Comment:
This will fail if(!result.hasDefined(Util.REQUEST_PROPERTIES)). An undefined node throws IAE if keys() is called.